build.sh

```diff
@@ -15,6 +15,22 @@ unset CALLBACK
 
 # Run build process based on configuration files
 
+# load nvm
+. "$NVM_DIR/nvm.sh"
+
+# use .nvmrc if it exists
+if [[ -f .nvmrc ]]; then
+  nvm install >&   /dev/null
+  nvm use >& /dev/null
+fi
+
+# install from package.json if it exists
+# run the federalist command
+if [[ -f package.json ]]; then
+  npm install >& /dev/null
+  npm run federalist || true >& /dev/null
```

Comment:
@shawnbot What do you think of having a `federalist` script that could be run instead of having to shell out from Ruby?